internal/dag/gatewayapi_processor.go

```diff
@@ -1145,14 +1175,24 @@ func (p *GatewayAPIProcessor) computeHTTPRouteForListener(route *gatewayapi_v1be
 
 		// Process rule-level filters.
 		var (
+			err                error
+			redirect           *Redirect
+			urlRewriteHostname string
+
+			retryPolicy          *RetryPolicy
+			mirrorPolicies       []*MirrorPolicy
 			requestHeaderPolicy  *HeadersPolicy
 			responseHeaderPolicy *HeadersPolicy
-			redirect             *Redirect
-			mirrorPolicies       []*MirrorPolicy
 			pathRewritePolicy    *PathRewritePolicy
-			urlRewriteHostname   string
+			requestTimeoutPolicy *RouteTimeoutPolicy
 		)
 
+		requestTimeoutPolicy, retryPolicy, err = parseHTTPRouteTimeouts(rule.Timeouts)
+		if err != nil {
+			routeAccessor.AddCondition(gatewayapi_v1beta1.RouteConditionResolvedRefs, metav1.ConditionFalse, status.ReasonInvalidTimeout, err.Error())
```

Comment:
lets add some tests for this added status condition for completeness

also I'm not sure we want to use the ResolvedRefs condition, I think we probably want to use the Programmed condition instead since we're actively not going to program this route rule given the `continue` below

I could even see an argument for setting `Accepted` to false given this would mean the route is not syntactically valid